app/views/articles/_form.html.erb

```diff
@@ -1,23 +1,46 @@
-<%= form_with model: article do |form| %>
-  <div>
-    <%= form.label :title %><br>
-    <%= form.text_field :title %>
-    <% article.errors.full_messages_for(:title).each do |message| %>
-      <div><%= message %></div>
-    <% end %>
-  </div>
-  <div>
-    <%= form.label :body %><br>
-    <%= form.text_area :body %><br>
-    <% article.errors.full_messages_for(:body).each do |message| %>
-      <div><%= message %></div>
-    <% end %>
-  </div>
-  <div>
-    <%= form.label :status %><br>
-    <%= form.select :status, %w[public private archived], selected: 'public' %>
-  </div>
-  <div>
-    <%= form.submit %>
-  </div>
-<% end %>
+<div class="container">
+    <div class="row">
+      <div class="col-lg-8 col-md-10 mx-auto">
+        <%= form_with model: article do |form| %>
+          <div class="control-group">
+            <div class="form-group floating-label-form-group controls">
+              <label>Title</label>
+              <%= form.text_field :title,
+                                  class: "form-control",
+                                  placeholder: "Title" %>
+              <% article.errors.full_messages_for(:title).each do |message| %>
+                <div><%= message %></div>
+              <% end %>
+              <p class="help-block text-danger"></p>
+            </div>
+          </div>
+          <div class="control-group">
+            <div class="form-group floating-label-form-group controls">
+              <label>Body</label>
+              <%= form.text_area :body,
+                                  class: "form-control",
+                                  placeholder: "Body",
+                                  rows: "5" %>
+              <% article.errors.full_messages_for(:body).each do |message| %>
+                <div><%= message %></div>
+              <% end %>
+              <p class="help-block text-danger"></p>
+            </div>
+          </div>
+          <div class="control-group">
+              <label>Status: </label><br>
+              <%= form.select :status,
+                              %w[public private archived],
+                              selected: article.status %>
+          </div>
+          <br>
+          <div>
+            <%= form.submit 'Send', :class => "btn btn-outline-primary"%>
```

Comment:
Remember to this syntax `class: 'btn btn-outline-primary'`.

Reply:
Thanks !